pom.xml

```diff
@@ -609,7 +621,30 @@
                                     </commandlineArgs>
                                 </configuration>
                             </execution>
-
+                            <execution>
+                                <id>RollCycleMultiThreadStressTest</id>
+                                <phase>test</phase>
+                                <goals>
+                                    <goal>exec</goal>
+                                </goals>
+                                <configuration>
+                                    <executable>${java.home}/bin/java</executable>
+                                    <commandlineArgs>${jvm.requiredArgs} -Djvm.resource.tracing=false -DtestTime=30 -classpath %classpath net.openhft.chronicle.queue.impl.single.stress.RollCycleMultiThreadStressTest
```

Comment:
Will it work like a benchmark e.g. expose data for TC to visualise?

Reply:
Yes